packages/core/src/types/IViewport.ts

```diff
@@ -122,20 +138,26 @@ export interface ViewReference {
    */
   viewUp?: Point3;
   /**
-   * The slice index or range for this view.
-   * <b>NOTE</b> The slice index is relative to the volume or stack of images.
-   * You cannot apply a slice index from one volume to another as they do NOT
+   * The slice index for the image of interest
+   * <b>NOTE</b> The slice index is relative to the volume or video image.
+   * You cannot apply a slice index from one volume or stack to another as they do NOT
    * apply.   The referencedImageId should belong to the volume you are trying
    * to apply to, the viewPlane normal should be identical, and then you can
    * apply the sliceIndex.
    *
-   * For stack viewports, the referencedImageId should occur at the given slice index.
+   * For stack viewports, the referencedImageId should be preferred
    *
    * <b>Note 2</b>slice indices don't necessarily indicate anything positionally
    * within the stack of images - subsequent slice indexes can be at opposite
    * ends or can be co-incident but separate types of images.
    */
-  sliceIndex?: number | [number, number];
+  sliceIndex?: number;
+  /**
+   * An end of a slice range.  This is used to indicate the end of the slices
+   * where the sliceIndex is the first point.  This is a positive value if defined,
+   * so can be tested for `sliceRangeEnd`.
+   */
+  sliceRangeEnd?: number;
```

Comment:
same

Reply:
I'd rather have a separate numberr here - this one will be left as indices as it actually refers to specific slices in a consistent fashion, but even with that, the use of sliceIndex is consistent between single and ranges, so I think I'd like to leave it as a separate value.

Reply:
As discussed we will move off `number` and do a referencedEndId or something